src/utilities/util_dump.c

```diff
@@ -814,6 +847,321 @@ dump_all_records(WT_CURSOR *cursor, bool reverse, bool json)
     return (0);
 }
 
+/*
+ * dump_explore --
+ *     Dump data in an interactive fashion.
+ */
+static int
+dump_explore(WT_CURSOR *cursor, const char *uri, bool reverse, bool pretty, bool hex, bool json)
+{
+#define MAX_LIMIT 256
```

Comment:
If feel this is really problematic. The name is vague, and there is no check to see if it the macro is already defined also it is not undef`ed at the end of the function.
It is used in multiple places to indicate the limits on multiple but logically independent arrays.

Reply:
Not sure if f4e21cadec5526841818df0750baf25af80289b6 is better, let me know.